apps/x509.c

```diff
@@ -122,6 +122,7 @@ const OPTIONS x509_options[] = {
     {"purpose", OPT_PURPOSE, '-', "Print out certificate purposes"},
     {"pubkey", OPT_PUBKEY, '-', "Print the public key in PEM format"},
     {"modulus", OPT_MODULUS, '-', "Print the RSA key modulus"},
+    {"bundle", OPT_BUNDLE, '-', "Process multiple certificates"},
```

Comment:
I propose using a more clear, to-the-point name, e.g.
```suggestion
    {"multi", OPT_MULTI, '-', "Process multiple certificates"},
```

Reply:
Are there any objections to changing the option name?

Reply:
Meh.  Updated option name.